assets/app/views/edit/health-checks.html

```diff
@@ -0,0 +1,85 @@
+<default-header class="top-header"></default-header>
+<div class="wrap no-sidebar">
+  <div class="sidebar-left collapse navbar-collapse navbar-collapse-2">
+    <navbar-utility-mobile></navbar-utility-mobile>
+  </div>
+  <div class="middle">
+    <!-- Middle section -->
+    <div class="middle-section surface-shaded">
+      <div class="middle-container has-scroll">
+        <div class="middle-content">
+          <div class="container surface-shaded">
+            <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+            <div class="row">
+              <div class="col-md-12">
+                <alerts alerts="alerts"></alerts>
+                <div ng-show="!containers.length">Loading...</div>
+                <form ng-show="containers.length" name="form">
+                  <h1>Health Checks: {{name}}</h1>
+                  <div class="help-block">
+                    Container health is periodically checked using readiness and liveness probes.
+                  </div>
+                  <div class="learn-more-block">
+                    <a href="{{'application_health' | helpLink}}" target="_blank">Learn more <i class="fa fa-external-link" aria-hidden> </i></a>
+                  </div>
+                  <fieldset ng-disabled="disableInputs">
+                    <div ng-repeat="container in containers"
+                         ng-init="formName = container.name + '-form'">
+                      <h2 ng-if="containers.length > 1">Container {{container.name}}</h2>
+                      <h3>Readiness Probe</h3>
+                      <div class="help-block" ng-if="$first">
+                        A readiness probe determines if the container is ready to
+                        service requests. A failed readiness probe signals that a
```

Comment:
... is ready to handle requests

... failed readiness probe means

those two minor tweaks make it sound a *little* less technical